tests/tracking/fluent/test_fluent.py

```diff
@@ -1304,3 +1307,23 @@ def test_get_parent_run():
     assert parent_run.data.params == {"a": "1"}
 
     assert mlflow.get_parent_run(run_id) is None
+
+
+def test_active_run_thread_safety():
```

Comment:
Is it possible to add tests for `active_experiment_id` and `last_active_run_id`?